### PR TITLE
Create typical bldg updated to weather file and climate zone. Also included defaults.

### PR DESCRIPTION
**Pull request by jugonzal07**
Added climate zone look up for weather files and appropriate ways to add them to the OS model such that the measure can run in the OS application when using predefined geometry files.

### CODE REVIEW

File: lib/measures/CreateTypicalBuilding/README.md.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<%#= README.md.erb is used to auto-generate README.md. %>
```
Comment (lymereJ) [inline lib/measures/CreateTypicalBuilding/README.md.erb:1]: @jugonzal07 - Is this file needed?

File: lib/measures/CreateTypicalBuilding/measure.rb [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+                      'ASHRAELaboratory.osm',
+                      'ASHRAEHospital.osm',
+                      'ASHRAEHighriseApartment.osm',
+                      'ASHRAEFullServiceRestaurant.osm',
+                      'ASHRAECourthouse.osm',
+                      'ASHRAECollege.osm',
+
+    ]
+
+    HVAC_TYPES = ['Inferred',
```
Comment (lymereJ) [inline lib/measures/CreateTypicalBuilding/measure.rb:295]: @jugonzal07 - Weren't we going to push these to a module?

Comment (jugonzal07) [reply lib/measures/CreateTypicalBuilding/measure.rb]: Fixed and update tested on OS 1.7 Application. Worked just fine.

File: lib/measures/CreateTypicalBuilding/README.md.erb (hunk 1 above)
Comment (jugonzal07) [reply lib/measures/CreateTypicalBuilding/README.md.erb]: Nope-- it was just one of the files that gets autogenerated when creating measures from the application. Removed.

**Review by weilixu — APPROVED** [review]

File: lib/measures/CreateTypicalBuilding/resources/create_typical_resources.rb [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    'ASHRAELargeOffice.osm',
+                    'ASHRAELargeHotel.osm',
+                    'ASHRAELaboratory.osm',
+                    'ASHRAEHospital.osm',
+                    'ASHRAEHighriseApartment.osm',
+                    'ASHRAEFullServiceRestaurant.osm',
+                    'ASHRAECourthouse.osm',
+                    'ASHRAECollege.osm',
+
+  ]
```
Comment (lymereJ) [inline lib/measures/CreateTypicalBuilding/resources/create_typical_resources.rb:27]: Instead of having a straight dictionary could we perhaps just have "typical geomtry names". For example instead of "ASHRAESmallOffice.osm", could we have "Small Office"? I'm afraid that users won't have much clue as to what "ASHRAESmallOffice.osm" corresponds to, I feel like it's a bit more clear if we remove the ".osm" and the reference to ASHRAE.

**Review by lymereJ — CHANGES_REQUESTED** [review]
Looks great! I just left one small comment that I think we need to address before merging.

File: lib/measures/CreateTypicalBuilding/resources/create_typical_resources.rb (hunk 3 above)
Comment (lymereJ) [reply lib/measures/CreateTypicalBuilding/resources/create_typical_resources.rb:27]: Actually, this is already the case.

**Review by lymereJ — CHANGES_REQUESTED** [review]
See comment above.

**Comment by jugonzal07** [comment]
> See comment above.

Addressed.

**Review by lymereJ — APPROVED** [review]
